elf2cfetbl.c

```diff
@@ -1404,7 +1404,8 @@ int32 GetDstFilename(void)
 
 int32 OpenSrcFile(void)
 {
-    int RtnCode;
+    int       RtnCode;
+    char      TimeBuff[50];
```

Comment:
Do we have a buffer size macro?

Reply:
Nothing I could find built in.  Documentation says a minimum of 26.  I'll just get truncated (and it's just an info message) so I didn't think it was worth any additional effort.  If you'd prefer we could just delete the message.

Reply:
The maximum is locality based, which is likely why it's not defined since a locality could be added that requires more space.

Reply:
Let's leave it, though I wonder if we might want to define something using cmake in the future

Reply:
I don't consider it a configuration parameter.  Macro is fine but I wouldn't expose it outside this unit.